include/nodes/internal/FlowScene.hpp

```diff
@@ -106,33 +106,46 @@ class NODE_EDITOR_PUBLIC FlowScene
    * @brief Node has been created but not on the scene yet.
    * @see nodePlaced()
    */
-  void nodeCreated(Node &n);
+  void
+  nodeCreated(QtNodes::Node &n);
 
   /**
    * @brief Node has been added to the scene.
    * @details Connect to this signal if need a correct position of node.
    * @see nodeCreated()
    */
-  void nodePlaced(Node &n);
+  void
+  nodePlaced(QtNodes::Node &n);
```

Comment:
The class is defined within the `QtNodes` namespace. What is the point of adding the namespaces everywhere?

Reply:
Here I just followed the Clazy warning about that

Reply:
fully-qualified-moc-types
Warns when a signal, slot or invokable declaration is not using fully-qualified type names, which will break old-style connects and interaction with QML. 

So not a problem with the codebase itself, but since the back-end could communicate with QML could be a point

Reply:
I see, make sense, thanks for explanation.